step-definitions/core/vue.js

```diff
@@ -35,20 +32,67 @@ Given(/^I log in to evan.network using vue( with )?(\w+)?$/, async (customPart,
   }, [], function(result) {
     this.assert.ok(result.value);
   });
+})
+
+
+Given(/^I log in to evan.network using vue( with )?(\w+)?$/, async (customPart, accountName) => {
+  const evan = setupEvan(client);
+
+
+
+  await client.url(`${ evan.baseUrl }#/dashboard.vue.evan`);
+  await client.execute(function() {
+    window.localStorage.setItem('evan-vault', '');
+    window.localStorage.setItem('evan-test-mode', true);
+    window.localStorage.setItem('evan-warnings-disabled', '{"payment-channel":true}');
+    window.localStorage.setItem('evan-language', 'en');
+    window.localStorage.setItem('evan-test-recaptchaId', '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI');
+    return true;
+  })
+  await client.pause(3000);
+  if (customPart && !evan.accounts[accountName]) {
+    throw new Error(`no account data found for account ${accountName}`);
+  }
+  const user = evan.accounts[accountName || 'default'] || evan.accounts.default;
```

Comment:
isn't `evan.accounts[accountName || 'default']` the same as `evan.accounts.default` if accountName is undefined.